templates/web/borsetshire/footer_extra_js.html

```diff
@@ -2,4 +2,5 @@
     version('/cobrands/fixmystreet-uk-councils/js.js'),
     version('/cobrands/borsetshire/google_maps_style.js'),
     version('/cobrands/borsetshire/js.js'),
+    version('/cobrands/fixmystreet/assets.js'),
```

Comment:
I think you only want to be adding this on map pages, see other cobrands, otherwise it throws a JS error on other pages.

Reply:
I'm not clear why this addition was needed, either.

Reply:
It's there because otherwise the stopper code isn't included and I can't test the bug :)

Reply:
Oh, so, should we think about having this code factored out and be able to be included without assets? Seems odd if it's possible to use entirely non-asset related, and people can set "disable the form" in the admin but wouldn't know this had to be included... Something to ticket anyway.